src/commands/friends/FriendCommand.ts

```diff
@@ -126,9 +126,34 @@ export default class FriendCommand extends Command {
     }
   }
 
-  async denyFriendRequest (msg: CommandMessage, user: User | string): Promise<Message | Message[]> {
-    // TODO: Delete friend request using API.
-    return msg.reply('This command is not ready yet.')
+  async denyFriendRequest(
+    msg: CommandMessage,
+    user: User | string | BotUser
+  ): Promise<Message | Message[]> {
+    if (msg.author.id === (user instanceof User ? user.id : user)) {
+      return msg.reply('You can\'t delete a friend request from yourself. That would be silly.')
+    }
+
+    if (!(user instanceof User)) {
+      const { data }: { data: BotUser } = await axios.get(`${Plugin.config.api.address}/users/${user}?token=${Plugin.config.api.token}`)
+      user = data
```

Comment:
Don't reassign parameters. Make a local variable to store the api user.
Also, use the function you created to get the api user.